restler/engine/core/sequences.py

```diff
@@ -777,6 +806,12 @@ def send_and_parse(request_data):
                 request_utilities.call_response_parser(request_data.parser, None, responses=responses_to_parse)
             return response.status_code
 
+        # TODO: when replaying from the trace DB is supported, the combination ID should be available, since
+        # the sequence object will be available.  It may also be useful to provide the original combination ID
+        # for the sequence, so that the replay can be associated with the original rendering.
```

Comment:
Not for this PR, but once this feature is supported it would be helpful if the original origin (e.g., "main_driver") could be recorded as a tag as well.